From: David Banks <amoebae@gmail.com>
Date: Sun, 12 May 2013 11:10:44 +0100
Subject: Remove redundant linking of vorbis libraries

Makefile attempts to link against libvorbis and libogg when using ogg support,
but these are not required, only libvorbisfile.  This avoids an unnecessary
dependency.

Origin: vendor, Debian
---
 Quake/Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Quake/Makefile
+++ b/Quake/Makefile
@@ -133,11 +133,11 @@
 endif
 ifeq ($(VORBISLIB),vorbis)
 cpp_vorbisdec=
-lib_vorbisdec=-lvorbisfile -lvorbis -logg
+lib_vorbisdec=-lvorbisfile
 endif
 ifeq ($(VORBISLIB),tremor)
 cpp_vorbisdec=-DVORBIS_USE_TREMOR
-lib_vorbisdec=-lvorbisidec -logg
+lib_vorbisdec=-lvorbisidec
 endif
 
 CODECLIBS  :=
